daemon/daemon_windows.go

```diff
@@ -485,6 +489,12 @@ func (daemon *Daemon) setDefaultIsolation() error {
 			if containertypes.Isolation(val).IsHyperV() {
 				daemon.defaultIsolation = containertypes.Isolation("hyperv")
 			}
+			if containertypes.Isolation(val).IsProcess() {
```

Comment:
TBH both are equally valid. I prefer it here as strictly the option chosen is a valid recognised option for isolation which is what that function is checking. I don't mind either way though, just as I say prefer it here.